app/controllers/examples_controller.rb

```diff
@@ -1,31 +1,31 @@
+# frozen_string_literal: true
+
```

Comment:
Lets not touch *examples* files because they are [autogenerated](https://github.com/la-ruby/create-rails-app/blob/f0641846a0fc331f7514d8ce01ddcfbeabee287a/create-rails-app#L307) in the parent repo